### PR TITLE
redis password

### PR DESCRIPTION
**Pull request by giggsoff**
We move to public clouds, so it is reasonable to set passwords where we can. 

Signed-off-by: Petr Fedchenkov <giggsoff@gmail.com>

### CODE REVIEW

**Comment by rvs** [comment]
This seems like a sensible thing to do ;-)

File: pkg/defaults/defaults.go [hunk 1]
```diff
@@ -62,10 +62,12 @@ const (
 	DefaultEveRegistry          = "lfedge"
 	DefaultRegistry             = "docker.io"
 
-	DefaultSFTPUser     = "user"
-	DefaultSFTPPassword = "password"
+	DefaultSFTPUser      = "user"
```
Comment (mydatascience) [inline pkg/defaults/defaults.go:65]: let's use something harder as a default. Many bruteforcers are guessing that combo. Also it's a rule of thumb that if you have the default then 95% of users will use that default.

File: pkg/utils/params.go [hunk 2]
```diff
@@ -19,3 +21,16 @@ func GetParams(line, regEx string) (paramsMap map[string]string) {
 	}
 	return
 }
+
+//GeneratePassword returns string with defined length and random characters
+func GeneratePassword(length int) string {
```
Comment (mydatascience) [inline pkg/utils/params.go:26]: We generate redis pass why don't we use the same for sftp pass?

Comment (giggsoff) [reply pkg/utils/params.go:26]: I think, it is good point for the future PRs. This PR is about redis password

Comment (mydatascience) [reply pkg/utils/params.go:26]: ok

**Pull request merged**